lib/resource.js

```diff
@@ -74,7 +77,17 @@ module.exports = class PodletClientResource {
             false,
         );
 
-        return this.resolver.resolve(outgoing).then(obj => obj.content);
+        return this.resolver.resolve(outgoing).then(obj => ({
+            content: obj.content,
+            js:
+                typeof obj.manifest.assets === 'object'
+                    ? obj.manifest.assets.js
+                    : '',
+            css:
+                typeof obj.manifest.assets === 'object'
+                    ? obj.manifest.assets.css
+                    : '',
+        }));
```

Comment:
@trygve-lie what should be returned if js and/or css are not available in the manifest file? undefined? null? empty string? empty array?

Reply:
Empty array.

Reply:
Have gone with empty string until we tackle https://github.com/podium-lib/issues/issues/10 at which point it needs to be changed to an empty array

Reply:
:+1: